src/Microsoft.AspNetCore.Razor.Runtime/project.json

```diff
@@ -56,9 +56,12 @@
     },
     "netstandard1.3": {
       "dependencies": {
+        "System.Collections.Concurrent": "4.0.12-*",
+        "System.Reflection": "4.1.0-*",
```

Comment:
Talked to @davidfowl and this should be bumped up. There's no good way to know whether these packages are compatible. This is supposed to all be fixed for RTM, but for now we shouldn't mix.